examples/model_selection/plot_validation_curve.py

```diff
@@ -49,3 +49,29 @@
                  color="navy", lw=lw)
 plt.legend(loc="best")
 plt.show()
+
+#####################################################################
+# The same plot can also be generated using a combination of GridSearchCV and
+# plotting module of scikit-learn.
+
+print(__doc__)
```

Comment:
This line will print again the docstring at the top of the file.
We can remove it